.github/workflows/integration_test.yml

```diff
@@ -8,41 +8,31 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        kube_provider: [kind]
+        cluster_provider: [kind,microshift]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
           go-version: 1.18
 
-      - name: use kepler action for kind cluster build
-        uses: sustainable-computing-io/kepler-action@v0.0.0
+      - name: use kepler action for cluster build
+        uses: sustainable-computing-io/kepler-action@main
         with:
-          runningBranch: kind
+          runningBranch: ${{matrix.cluster_provider}}
+          local_dev_cluster_version: main
+          cluster_provider: ${{matrix.cluster_provider}}
 
       - name: simple test - deploy kepler
         run: make cluster-sync
         env:
-            CLUSTER_PROVIDER: ${{matrix.kube_provider}}
-            IMAGE_REPO: "localhost:5001"
-            IMAGE_TAG: "devel"
-            CTR_CMD: docker
-            CI_ONLY: "true"
+          CLUSTER_PROVIDER: ${{matrix.cluster_provider}}
+          IMAGE_REPO: "localhost:5001"
+          IMAGE_TAG: "devel"
+          CTR_CMD: docker
+          CI_ONLY: "true"
 
-      - name: test if kepler is still alive
```

Comment:
I hope we can keep this part of check. As for historical reason,  kepler is not started correctly, hence this part is to check if kepler pod started.

Reply:
Yes, @SamYuan1990 I have added another [script](https://github.com/sustainable-computing-io/kepler/blob/92722eb34559bd48f75edc19ac32018fc1fafa55/hack/verify.sh) inside the hack folder which will test if kepler is still alive as well as launch e2e tests. It is called from [here](https://github.com/sustainable-computing-io/kepler/blob/92722eb34559bd48f75edc19ac32018fc1fafa55/hack/cluster-deploy.sh#L48)

Reply:
ok